packages/create-toolpad-app/src/index.ts

```diff
@@ -87,116 +65,136 @@ async function isFolderEmpty(root: string, name: string): Promise<boolean> {
 // Detect the package manager
 const packageManager = getPackageManager();
 
-// Install the dependencies
-const installDeps = async (projectName: string, cwd: string) => {
-  const { execaCommand } = await import('execa');
+const validatePath = async (relativePath: string): Promise<boolean | string> => {
+  const { default: chalk } = await import('chalk');
+
+  const absolutePath = path.join(process.cwd(), relativePath);
+
+  try {
+    await fs.mkdir(absolutePath);
+
+    return true;
+  } catch {
+    // Directory exists, verify if it's empty to proceed
+    try {
+      if (await isFolderEmpty(absolutePath)) {
+        return true;
+      }
+      return `${chalk.red('error')} - The directory at ${chalk.blue(
+        absolutePath,
+      )} contains files that could conflict. Either use a new directory, or remove conflicting files.`;
+    } catch {
+      return `${chalk.red('error')} - Unable to access directory at ${chalk.red(
```

Comment:
This way there is no feedback to the user about what went wrong exactly, since it's unexpected, I'd probably just let the error bubble up and crash the program.